gradle/versions.gradle

```diff
@@ -11,7 +11,7 @@ ext.libVersions = [
     jackson: '2.5.1',
     jackson_annotation: '2.5.0',
     cassandra: '2.2.1',
-    cassandra_driver_core: '2.2.0-rc3',
+    cassandra_driver_core: '2.1.5',
```

Comment:
We should also look at the 3.0.0 driver which still has compatibility with C* 1.2.x, see https://github.com/datastax/java-driver/blob/3.0.0/README.md